sbin/build.sh

```diff
@@ -20,7 +20,13 @@ WORKING_DIR=$1
 TARGET_DIR=$2
 OPENJDK_REPO_NAME=$3
 BUILD_FULL_NAME=$4
-JVM_VARIANT=${5:-normal}
+JVM_VARIANT=${5:=normal}
+NOBUILD=$6
```

Comment:
I also noticed the use of `:=` instead of `:-` (for default values), the difference is minor although we should agree on which/what to always use.